ext/rotoscope/rotoscope.c

```diff
@@ -240,20 +237,45 @@ static Rotoscope *get_config(VALUE self)
   return config;
 }
 
+void copy_blacklist(Rotoscope *config, VALUE blacklist) {
+  size_t blacklist_malloc_size = RARRAY_LEN(blacklist) * sizeof(*config->blacklist);
+
+  Check_Type(blacklist, T_ARRAY);
+  for (long i = 0; i < RARRAY_LEN(blacklist); i++) {
```

Comment:
For some reason `RARRAY_LEN` returns a `long` value, but we use `unsigned long` for `config->blacklist_size`